proto/backend.proto

```diff
@@ -200,8 +200,9 @@ message SubscribeStreamRequest {
 
   // path part of channel.
   string path = 2;
-  // optional raw data query.
-  bytes query = 3;
+  // optional raw data. May be used as an extra payload supplied upon subscription.
+  // For example, can contain JSON query object.
+  bytes data = 3;
```

Comment:
Since this is only a rename of the field it shouldn't be a breaking change with protobuf, correct?

Reply:
Also, just a rename of new field which is not part of `main` yet.